GPU/GPUTracking/CMakeLists.txt

```diff
@@ -334,7 +334,19 @@ if(ALIGPU_BUILD_TYPE STREQUAL "O2")
     ${targetName}
     PRIVATE $<TARGET_PROPERTY:O2::Framework,INTERFACE_INCLUDE_DIRECTORIES>)
 
-  target_compile_definitions(${targetName} PRIVATE GPUCA_O2_LIB GPUCA_TPC_GEOMETRY_O2 GPUCA_HAS_ONNX=1)
+  message("Compile definitions for ONNX runtime:")
+  message(STATUS "ORT_ROCM_BUILD: ${ORT_ROCM_BUILD}")
+  message(STATUS "ORT_CUDA_BUILD: ${ORT_CUDA_BUILD}")
+  message(STATUS "ORT_MIGRAPHX_BUILD: ${ORT_MIGRAPHX_BUILD}")
+  message(STATUS "ORT_TENSORRT_BUILD: ${ORT_TENSORRT_BUILD}")
+
+
+  target_compile_definitions(${targetName} PRIVATE
+    GPUCA_O2_LIB GPUCA_TPC_GEOMETRY_O2 GPUCA_HAS_ONNX=1
```

Comment:
Why do you need the GPU specifix settings for the general GPUTracking library? Can't you put them only into the backend libraries?